contrib/container-build/buildit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+OUTDIR=~/build/out
```

Comment:
Let's get the binaries out to the same directory that it would've gotten out at, that way we also will retain the architecture and compiler information.

Reply:
Done and done